docs/installation.md

```diff
@@ -14,12 +21,12 @@ separately:
 
```

Comment:
Good question.  I wasn't aware that catkin made it into the Ubuntu repos so that it could actually be installed without ROS.  I'm not sure if we have any actual dependency on ROS packages.